Dockerfile

```diff
@@ -36,4 +36,6 @@ RUN COMMIT=`git rev-parse HEAD` && echo "COMMIT=\"${COMMIT}\"" >$appname/version
 
 WORKDIR /var/www/$appname
 
-CMD /dockerrun.sh
+ENTRYPOINT [ "/bin/sh", "/dockerrun.sh" ]
```

Comment:
Just curious--why?

Reply:
no particular reason - I was using this for testing, Entrypoint makes it an executable docker image and will always be executed on docker run. I don't think this is actually needed, I'll change it back to CMD to be consistent with fence, sheepdog behavior.

Reply:
Cool cool!